LayoutTests/imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-animation-and-scroll.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<title>Test that making a content-visibility: hidden subtree visible restarts animations in it</title>
+<link rel="author" title="Rob Buis" href="mailto:rbuis@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-contain-2/">
+<link rel="match" href="content-visibility-animation-becomes-visible-ref.html">
+<meta name="assert" content="animations in content-visibility: hidden subtree should restart once visible">
+
+<script src="/common/reftest-wait.js"></script>
+<script src="../resources/utils.js"></script>
+
+<style>
+#container {
+  content-visibility: auto;
+  contain-intrinsic-size: 100px 100px;
+}
+@keyframes fade {
+  from { opacity: 1; }
+  to { opacity: 0;  }
+}
+#target {
+  background: green;
+  height: 100px;
+  width: 100px;
+}
+.animate {
+  animation: fade 1s linear 2 alternate;
+  animation-delay: 1s;
```

Comment:
This delay doesn't seem correct to me. You'd screenshot an animation that never started as a result IIUC.

It would be good to have the animation go from transparent to green and then screenshot once the animation ended (e.g. 1s later). That should accurately assert that the animation successfully re-started.

`animation-fill-mode: forwards;` should help taking screenshot a stable state.

Reply:
Would also be good if the animation involved some acceleration in addition to opacity too (transform for instance)

Reply:
Fair points, no more delay and I added a transform.